tests/conftest.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+import threading
+import traceback
+import socket
+
+from server import ConnectionListener
+
+# fmt: off
+motd_dict_test = {
+    "motd": [
+        "- Hello {user_nick}, this is a test MOTD!",
+        "-",
+        "- Foo",
+        "- Bar",
+        "- Baz",
+        "-",
+        "- End test MOTD"
+        ]
+}
+
+# fmt: on
+
+# Based on: https://gist.github.com/sbrugman/59b3535ebcd5aa0e2598293cfa58b6ab#gistcomment-3795790
+@pytest.fixture(scope="function")
```

Comment:
`scope="function"` is the default, and doesn't need to be specified:

```suggestion
@pytest.fixture
```